modules/ROOT/pages/create-map-json-xml-to-outbound-x12 copy.adoc

```diff
@@ -0,0 +1,68 @@
+= Create a Map For XML or JSON to Outbound EDI X12
+
+Follow these steps to create the DataWeave map to transform your application message formats such as XML or JSON to outbound EDI X12 transactions:
+
+. Launch Anypoint Studio.
+. From the Anypoint Studio *File* menu, select *New > Project from Template*.
+. Log in to Anypoint Exchange with your Anypoint platform credentials.
+. Search for `X12`.
+. Click the template named `Template to map application messages to outbound EDI X12 transactions`.
+. Click *Open*, to create a new project in your Studio workspace pre-loaded with the template. 
+
+== Set the Source Message Type
+
+To set the source JSON or XML message type for the map:
+
+. Copy the schema file of your target application message type into the `src/main/resources` folder in the template project.
+. From the *Mule Palette* view, click the *Transform Message* component and drag it on to the Studio canvas.
+. Select the *Transform Message* component and in the input section, next to *Payload*, click *Define metadata*.
++
+image::xml-to-outbound-x12-payload-add-metadata.png[Transform Message component add payload metadata]
++
+. Click *Add*.
+. In *Type id*, enter a new message type ID, for example: `enterprise-outbound-purchase-order`.
+. Click *Create type*.
+. In the *Select metadata type* dialog window, from the *Type* drop-down menu, select *JSON* or *XML*. 
+. Browse to the location of the JSON or XML schema file you imported earlier and click *Select*.
+
+== Set the Target X12 Message Type
+
+The template has the X12 configuration for the below X12 transaction set:
+
+* Version +
+`4010`
+* Transaction Set + 
+`850 (Purchase order)`
+
+Follow these steps to set the target X12 message type:
+
+. Click the *Global Elements* tab at the base of the Studio canvas in `global-configs.xml`.
+. From *Global Configuration Elements*, select *X12 EDO Config* and click *Edit*.
+. In *Name*, change the name of the configuration.
+. In the *Schema definitions*, select *Edit inline*. 
+. Select the value */x12/004010/850.esl* and click the edit icon.
++
+image::xml-to-outbound-x12-edit-schema.png[X12 EDI Config dialog]
++
+. Change the Schema value to the version and transaction set you want to map from, for example, set the value to `/x12/004010/204.esl`, if you want to map to a 4010-204 Load tender transaction. +
+You now see your application message structure (JSON or XML) on the input pane in the left, and the X12 transaction set structure on the output pane in the right.
+. Click *Finish*, then click *OK*. +
+You can now begin mapping from input to the output.
+
+You can download a sample project from Exchange that has prebuilt mapping to transform outbound purchase order messages from XML to 4010 850 transactions.
```

Comment:
Why?
Also, reread that sentence to be sure it is structured to clearly indicate your meaning. It sounds pretty complicated to me....